src/modules/app/createReduxStore.js

```diff
@@ -33,7 +32,6 @@ const createReduxStore = (createHistory, initialState = {}, routes = new RouteCo
   }
 
   const reducer = combineReducers({
-    ...endpointReducers,
```

Comment:
Do we have any additional reducers/actions we need now because the models are no longer in the redux-state?